src/sentry/mediators/external_requests/select_requester.py

```diff
@@ -50,7 +50,15 @@ def _build_url(self):
 
     def _make_request(self):
         try:
-            body = safe_urlread(safe_urlopen(url=self._build_url(), headers=self._build_headers()))
+            body = safe_urlread(
+                send_and_save_sentry_app_request(
+                    self._build_url(),
+                    self.sentry_app,
+                    self.install.organization_id,
+                    "select.requested",
```

Comment:
@nolachen Nit: What about calling this `select_fields.requested`? I am concerned that `select.requested` might be a bit too ambiguous.

Reply:
@scefali would it be confusing in that the request is just for 1 select field? Maybe `select_field.requested` ? or like `select_options.requested`?

Reply:
@nolachen I like `select_options.requested`